ddtrace/contrib/sanic/__init__.py

```diff
@@ -0,0 +1,73 @@
+"""
+The Sanic__ integration will add tracing to all requests to your Sanic application.
+
+
+Enable Sanic tracing automatically via ``ddtrace-run``::
+
+    ddtrace-run python app.py
+
+Sanic tracing can also be enabled manually::
+
+    from ddtrace import patch_all
+    patch_all(sanic=True)
+
+    from sanic import Sanic
+    from sanic.response import text
+
+    app = Sanic(__name__)
+
+    @app.route('/')
+    def index(request):
+        return text('hello world')
+
+    if __name__ == '__main__':
+        app.run()
+
+
+Configuration
+~~~~~~~~~~~~~
+
+.. py:data:: ddtrace.config.sanic['distributed_tracing_enabled']
+
+   Whether to parse distributed tracing headers from requests received by your Sanic app.
+
+   Default: ``True``
+
+.. py:data:: ddtrace.config.sanic['analytics_enabled']
+
+   Whether to analyze spans for Sanic in App Analytics.
+
+   Can also be enabled with the ``DD_SANIC_ANALYTICS_ENABLED`` environment variable.
```

Comment:
```suggestion
   Can also be enabled with the ``DD_TRACE_SANIC_ANALYTICS_ENABLED`` environment variable.
```

this changed recently with #1575